packages/backend/src/clientKeys/routes.ts

```diff
@@ -37,10 +39,8 @@ export async function getKeyById(
     key: key.jwk,
     client: {
```

Comment:
do we want update the response body on this as well? what do we need this route for btw?

Reply:
Key inspection from auth server I guess? I updated the body.